### DISCUSSION TITLE
Provide suffices for amount of cycles

### DISCUSSION THREAD

**Discussion by pakhomov-dfinity** [dfinity/sdk#3011] [Feature Requests]
E.g. `dfx canister --network=testnet deposit-cycles 10G bitcoin.` Counting zeros is an unpleasant user experience.

**Comment by ghost** [comment]
Implemented in https://github.com/dfinity/sdk/pull/3452 and available in dfx 0.15.2

> Certain suffixes that replace a number of zeros are now supported. The (case-insensitive) suffixes are:
> - `k` for `000`, e.g. `500k`
> - `m` for `000_000`, e.g. `5m`
> - `b` for `000_000_000`, e.g. `50B`
> - `t` for `000_000_000_000`, e.g. `0.3T`

> For cycles an additional `c` or `C` is also acceptable. For example: `dfx canister deposit-cycles 3TC mycanister`
